datasets/scene_parse_150/README.md

```diff
@@ -0,0 +1,224 @@
+---
+annotations_creators:
+- crowdsourced
+- expert-generated
+language_creators:
+- found
```

Comment:
well there's no language haha, once the YAML schema is updated we can probably remove that

Reply:
I use this tag to indicate from where the image data comes. Maybe we can rename it to `data_creators` later?